generators/app/index.js

```diff
@@ -94,28 +123,32 @@ module.exports = class extends BaseGenerator {
                     .slice(0, -1)
                     .join('.')
                     .match(/\w*$/i);
-                entitiesChoices.push({
-                    name: className,
-                    value: className
-                });
+                if (className) {
+                    entitiesChoices.push({
+                        name: className[0],
+                        value: className[0]
```

Comment:
You should put only once the `[0]` on the `className` variable declaration

Reply:
i want to test the result of the matcher in case of an undefined is returned before use it